util/limiter/limiter.go

```diff
@@ -0,0 +1,146 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+package limiter
+
+import (
+	_ "embed"
```

Comment:
Needed for anything?

Reply:
Nope, was an extra thing I decided to do separately for reasons. Removed.